lib/horde/registry.ex

```diff
@@ -34,34 +34,39 @@ defmodule Horde.Registry do
   Then you can use `MyRegistry.child_spec/1` and `MyRegistry.start_link/1` in the same way as you'd use `Horde.Registry.child_spec/1` and `Horde.Registry.start_link/1`.
   """
 
+  @type option ::
+          {:keys, :unique}
+          | {:name, registry()}
+          | {:delta_crdt_options, [DeltaCrdt.crdt_option()]}
+          | {:members, [Horde.Cluster.member()]}
+
   @callback init(options :: Keyword.t()) :: {:ok, options :: Keyword.t()}
+  @callback child_spec(options :: [option()]) :: Supervisor.child_spec()
```

Comment:
Adopted as is from [code base](https://github.com/Ajwah/horde/blob/0beaaacfea51f179e7e6e855c089077e270f165a/lib/horde/registry.ex#L89). Introduced because of:
> Elixir has always advocated that a behaviour must always be defined when a set of functions is marked as overridable
as stated by [Jose Valim](https://github.com/elixir-lang/elixir/issues/5734)

Reply:
:joy: I wrote that spec in Elixir.Registry, just got a PR for that accepted this week.

Reply:
Also, could you please add this to `Horde.Supervisor`?

Reply:
I will add it in a separate PR